tests/tensorflow/sparsity/rb/test_integration.py

```diff
@@ -141,7 +142,8 @@ def test_rb_sparse_target_lenet(distributed, quantized):
         if quantized:
             config.update({'compression': [config['compression'], {'algorithm': 'quantization'}]})
 
-        compress_algo, compress_model = create_compressed_model(model, config, should_init=False)
+        compression_state_to_skip_init = {BaseCompressionAlgorithmController.BUILDER_STATE: dict()}
+        compress_algo, compress_model = create_compressed_model(model, config, compression_state_to_skip_init)
```

Comment:
Had to hack in the test, because it doesn't run init and doesn't provide checkpoint.
We suppose, that customer will never do that: whether init or checkpoint should be used.